lib/inputs/os-release/static.ts

```diff
@@ -0,0 +1,92 @@
+import { getContentIfExists } from "../../extractor";
+import { ExtractAction, ExtractedLayers } from "../../extractor/types";
+import { streamToString } from "../../stream-utils";
+
+export { getOsReleaseActions, getOsRelease };
+
+const getOsReleaseAction: ExtractAction = {
+  actionName: "os-release",
+  fileNamePattern: "/etc/os-release",
+  callback: (stream) => streamToString(stream),
```

Comment:
Where we set the callback, can we pass the function directly e.g.: `callback: streamToString,`?

Reply:
I knew it looked suspicious to repeat it like that everywhere 😄Nice obvious fix, thanks; done